src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/unsubscribe_button_request.cc

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2018, Ford Motor Company
```

Comment:
Please change year

Reply:
fixed in 6cb2c51fcf2852a641cf1f556fc461c1f2a562fd